tests/testing_meshpy.py

```diff
@@ -541,6 +543,41 @@ def test_meshpy_geometry_sets(self):
         # Check the output.
         compare_test_result(self, mesh.get_string(header=False))
 
+    def test_unique_ordering_of_line_condition(self):
+        """
+        This test case ensures that the ordering of nodes defined within the line condition of the input file is unique.
+        """
+
+        # set up a beam mesh with material
+        input_file = InputFile()
+        mat = MaterialReissner(youngs_modulus=1.0, radius=1)
+        beam_set = create_beam_mesh_line(
+            input_file, Beam3rHerm2Line3, mat, [0, 0, 0], [2, 0, 0], n_el=10
+        )
+
+        # apply different dirichlet to all nodes within this condition
+        for i, node in enumerate(beam_set["line"].get_all_nodes()):
+
+            # do not constraint middle nodes
+            if not node.is_middle_node:
+                # add different condition value for each node
+                input_file.add(
+                    BoundaryCondition(
+                        GeometrySet(node),
```

Comment:
We can use a “dummy” string here for testing purposes.

Reply:
I used to coordinate in x direction, so that one can easily track which points are changing.